apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -268,6 +268,32 @@ const BookingPage = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const getBookingRedirectExtraParams = (booking: any) => {
+    if (!eventType.metadata?.passExtraParamsToRedirectUrl) {
+      return {};
+    }
+
+    const objectQueryParamKeys = ["customInputs", "responses", "metadata", "user", "attendees"];
```

Comment:
`customInputs` is deprecated(it is there only for old Event Types and is migrated to responses format when that even-type is saved again). So, we can remove it
```suggestion
    const objectQueryParamKeys = [ "responses", "metadata", "user", "attendees"];
```